dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/BaseClasses.mo

```diff
@@ -24,6 +24,7 @@ package BaseClasses
 
     type PStatus = enumeration(Standard "Active power is modulated by the frequency deviation", LimitPMin "Active power is fixed to its minimum value", LimitPMax "Active power is fixed to its maximum value");
 
+    Connectors.ImPin deltaPmRefPu(value(start = 0)) "Active power reference step in pu (base PNom)";
```

Comment:
Is this variable necessarily a step ? If not, I would write "Additional active power reference in pu (base PNom)".

By the way, shouldn't we call this a mechanical power (Pm) instead of an active power ?